client/src/components/matrix/MatrixCell.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./MatrixCell.css";
+import { createSelectable } from "react-selectable";
+
+class MatrixCell extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.Caseref = React.createRef();
+  }
+  componentDidMount() {
```

Comment:
In the future try to avoid using componentDidMount, componentDidUpdate etc. React hooks such as useEffect, handles components statutes just fine.